apps/web/app/[locale]/(home)/products/[productId]/components/add-to-card.tsx

```diff
@@ -0,0 +1,19 @@
+import { Button } from "@ikseer/ui/src/components/ui/button";
+import { LuShoppingCart } from "react-icons/lu";
+import useCart from "../../../store/cart/cart";
+
+// biome-ignore lint/suspicious/noExplicitAny: <explanation>
+export default function AddToCardButton({ product }: any) {
+	const { addItemToCart } = useCart();
+	const handleOnClick = () => {
+		addItemToCart(product);
+	};
+	return (
+		<Button
```

Comment:
We should use `react-query` instead of the global store created by Zustand here. React Query will make this global store for us with the ability to invalidate cached and refetching the updated from the backend. Also we can store and share the state with backend and between different devices that the user use.